plugins/woocommerce/client/blocks/assets/js/atomic/blocks/product-elements/image/block.json

```diff
@@ -48,7 +50,7 @@
 		"woocommerce/product-template",
 		"core/post-template"
 	],
-	"usesContext": [ "query", "queryId", "postId" ],
+	"usesContext": [ "imageId", "postId", "query", "queryId" ],
```

Comment:
In testing steps you mention:

> Add a new context provider to the providesContext object:
> `"woocommerce/product-image/imageId": "imageId"`

so either testing steps should be updated or update `usesContext`:

```suggestion
	"usesContext": [ "woocommerce/product-image/imageId", "postId", "query", "queryId" ],
```